source/arch/metadatastore-format.rst

```diff
@@ -171,9 +170,9 @@ above ``event`` is described by the ``event_descriptor`` ::
             "chan6" : {"source" : "PV:XF:23ID1-ES{Sclr:1}.S6"},
             "chan7" : {"source" : "PV:XF:23ID1-ES{Sclr:1}.S7"},
             "chan8" : {"source" : "PV:XF:23ID1-ES{Sclr:1}.S8"}, 
-            "pimte: : {"source" : "FILESTORE:<...>"}
+            "pimte: : {"source" : "name_of_detector", "external": "FILESTORE:<...>"}
```

Comment:
I think this document is definitely not concrete and always up for discussion. I wouldn't assume anything here is remotely solidified.